GUI.ipynb

```diff
@@ -93,7 +93,7 @@
    "outputs": [],
    "source": [
     "source = intake.gui.sources[0]\n",
-    "source.to_dask().head()"
+    "source().to_dask().head()"
```

Comment:
Hm, this should not be necessary ( maybe a consequence of https://github.com/intake/intake/pull/695 ??). To check: are the sources passed to SourceGUI.select entries or sources? 

Also, the GUI has a source_instance method which maybe would do this job without any change.

Reply:
I suppose the behaviour is OK, then, but the variable name is a little poor. Really we are selecting entries, and that's what gets displayed until you do something to it like plot; but the distinction is mainly hidden from the user.